core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4608,13 +4608,6 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey FUSE_LOGGING_THRESHOLD =
```

Comment:
prefer to keep it. it turned out useful

Reply:
I agree that it's useful. The reason that I remove it is because
(1) duplicate time recording one by metrics.timer and one by manually getting the start/end time.
(2) The functionalities can also be served by Timer metrics P75, P90, P95, P99. If the P90 number is larger than 10S, then 10% of this operation is taking longer 10s. Although I agree that the logging may be easier to be understand by end users.

Whats your thoughts @apc999？

Reply:
Ping @apc999

Reply:
it is often more useful when we know the time sequence in the log which helps debugging in my past experience.
e.g., knowing the location of expensive FUSE calls can help checkout the abnormals in other components.

Reply:
Make sense, will add the log back